### DISCUSSION TITLE
Is there a way to require `select` in queries?

### DISCUSSION THREAD

**Discussion by elie222** [prisma/prisma#14998] [Q&A]
For example, on the user object I don't want to accidentally return private information to the wrong user. This can be solved by using `select` and picking out the specific fields to return. But is there an easy way to require that select is always used? Or always use for a specific table?

**Comment by nurul3101** [comment]
Hi @elie222 👋 
It's not currently supported by Prisma yet. Could you create a [Feature Request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for it and explain your use case?

How do you envision this to work? Something like Prisma allows a configuration where you specify the model name and the fields which need to be selected from that model and all the queries of that particular model just return the fields specified in the configuration?

**Reply by vladi-stevanovic** [reply]
👋 Hello @elie222 I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can help you further or if you've already solved it, we'd love to know how! 

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
